packages/web-components/src/components/button-group/button-group-item.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { customElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import styles from './button-group.scss';
+import DDSBtn from '../button/button';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Button group item.
+ *
+ * @element dds-button-group-item
+ */
+@customElement(`${ddsPrefix}-button-group-item`)
+class DDSButtonGroupItem extends DDSBtn {
+  static styles = styles; // `styles` here is a `CSSResult` generated by custom WebPack loader
+
+  static get stableSelector() {
+    return `${ddsPrefix}--button-group-item`;
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'listitem');
+    }
+  }
```

Comment:
Style nit, the rest of the codebase put `static` fields/methods the bottom:

```suggestion
  connectedCallback() {
    super.connectedCallback();
    if (!this.hasAttribute('role')) {
      this.setAttribute('role', 'listitem');
    }
  }

  static get stableSelector() {
    return `${ddsPrefix}--button-group-item`;
  }

  static styles = styles; // `styles` here is a `CSSResult` generated by custom WebPack loader
```